### DISCUSSION TITLE
anisotropic filtering in ImageCompose

### DISCUSSION THREAD

**Discussion by tgd-git** [cables-gl/cables_docs#256] [Feature Requests]
it would be cool to have this as option like in the texture op. is it possible?

**Comment by positlabs** [comment]
Image compose only works on 2d textures, right? I think anisotropic filtering requires some information about the 3d geometry to work. Maybe you are looking for something like anti-aliasing instead?
<img width="812" alt="Screen Shot 2022-06-01 at 11 56 22 AM" src="https://user-images.githubusercontent.com/1918844/171448066-a0656c59-a77f-4383-917a-983b7e9eb32b.png">

**Comment by tgd-git** [comment]
Please see the texture on the right rectangle is crisp in the distance (texture from texture op) and the left one is blurry (texture from image compose op) 
![image](https://user-images.githubusercontent.com/2562637/171488581-837d9c8d-ed46-4292-b3b0-956e44eda224.png)
